src/runners/handlers/slack.py

```diff
@@ -74,30 +73,46 @@ def handle(
     title = alert['TITLE']
 
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
-
-        # log.info(f'Slack user info for {email}', result)
-
-        if result['ok'] is True and 'error' not in result:
-            user = result['user']
-            userid = user['id']
-        else:
-            log.error(f'Cannot identify  Slack user for email {recipient_email}')
-            return None
-
+        if isinstance(recipient_email, str):
+                user = sc.api_call("users.lookupByEmail", email=recipient_email)
+                if not user['ok']:
+                    log.error(f'Cannot identify Slack user for email {recipient_email}')
+                    return None
+                    
+                else:
+                    userid = user['user']['id']
+                    result = sc.api_call("conversations.open", users=userid)
+                    user_id = result['channel']['id']
+                    if not result['ok']:
+                        log.error(f'Error ocurred while opening conversation channel')
+                        return None
+                        
+        elif(recipient_email, list):
+            users = []
+            for email in recipient_email:
+                user = sc.api_call("users.lookupByEmail", email=email)
+                if not user['ok']:
+                    log.error(f'Cannot identify Slack user for email {email}')
+                    return None
+                users.append(user['user']['id'])
+                user_ids = ",".join(users)                  
+                #converting list to comma seperated string
+            result = sc.api_call("conversations.open", users=user_ids)
+            user_id = result['channel']['id']
+            
     # check if channel exists, if yes notification will be delivered to the channel
     if channel is not None:
         log.info(f'Creating new SLACK message for {title} in channel', channel)
     else:
         if recipient_email is not None:
-            channel = userid
+            channel = user_id
```

Comment:
I'm not sure what this rename serves, but if you rename it channel_id above, it seems to make sense to keep that name throughout.